src/recepResponder/java/uk/nhs/digital/nhsconnect/nhais/ResponderNhaisApplication.java

```diff
@@ -21,7 +21,7 @@ public static void main(String[] args) {
 
 		@Bean
 		String registrationConsumerService() {
-			return RecepResponderService.class.getSimpleName();
+			return uk.nhs.digital.nhsconnect.nhais.responder.RecepResponderService.class.getSimpleName();
```

Comment:
why here is the whole path? Why not to import it like it was before?

Reply:
It's IntelliJ refactor after I moved `ResponderNhaisApplication` from `uk.nhs.digital.nhsconnect.nhais.responder` to `uk.nhs.digital.nhsconnect.nhais`